### PR TITLE
BuzzFeed News & AskUbuntu

### PR DESCRIPTION
**Pull request by Lanson2232**
**Reading a Article :**
Discord_HXa95o50xx

**Viewing To Section(All Sections Supported) :**
Discord_VmQ4ULC9GB

**Viewing To User Profile :**
Discord_PW1W1orGP4

### CODE REVIEW

File: BuzzFeedNews/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+"service": "BuzzFeed News",
+"description": {
+"en": "BuzzFeed, Inc. is an American Internet media, news and entertainment company with a focus on digital media; it is based in New York City. BuzzFeed was founded in 2006 by Jonah Peretti and John S. Johnson III, to focus on tracking viral content.",
+"tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu."
+},
+"url": [
+"www.buzzfeednews.com",
+"buzzfeednews.com"
+],
+"version": "1.3",
```
Comment (eggsywashere) [inline BuzzFeedNews/dist/metadata.json]: Please change the version number every time you make an update.

Comment (Lanson2232) [reply BuzzFeedNews/dist/metadata.json]: 👌

File: AskUbuntu/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+"en": "Ask Ubuntu is a community driven question and answer website for the Ubuntu operating system. It is part of the Stack Exchange Network, running the same software as Stack Overflow. Members gain reputation based on the community's response to their questions and answers.",
+"tr": "Ask Ubuntu, Ubuntu işletim sistemi için  topluluk odaklı bir soru cevap web sitesidir. Stack Overflow ile aynı yazılımı çalıştıran Stack Exchange Network'ün bir parçasıdır. Üyeler, topluluğun sorularına ve cevaplarına verdiği yanıtlara dayanarak itibar kazanır."
+},
+"url": [
+"www.askubuntu.com",
+"askubuntu.com"
+],
+"version": "1.0.1",
+"logo": "https://cdn.sstatic.net/Sites/askubuntu/img/apple-touch-icon@2.png",
+"thumbnail": "https://cdn.discordapp.com/attachments/647557097701244953/651454166853615616/opera_wabv8Za7Xe.png",
```
Comment (Bas950) [inline AskUbuntu/dist/metadata.json]: please use another website to upload the images to, not discords cdn

File: BuzzFeedNews/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+"en": "BuzzFeed, Inc. is an American Internet media, news and entertainment company with a focus on digital media; it is based in New York City. BuzzFeed was founded in 2006 by Jonah Peretti and John S. Johnson III, to focus on tracking viral content.",
+"tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu."
+},
+"url": [
+"www.buzzfeednews.com",
+"buzzfeednews.com"
+],
+"version": "1.4",
+"logo": "https://cdn.discordapp.com/attachments/647557097701244953/651449880853741568/BuzzFeedNews.png",
+"thumbnail": "https://cdn.discordapp.com/attachments/647557097701244953/651449931202297869/LvqYqJ.png",
```
Comment (Bas950) [inline BuzzFeedNews/dist/metadata.json]: same here

File: AskUbuntu/dist/metadata.json (hunk 2 above)
Comment (Lanson2232) [reply AskUbuntu/dist/metadata.json]: done 👌

File: BuzzFeedNews/dist/metadata.json (hunk 3 above)
Comment (Lanson2232) [reply BuzzFeedNews/dist/metadata.json]: done 👌

**Comment by Lanson2232** [comment]
**AskUbuntu Usage ScreenShots**

User Profile:
UserProfile

Users page:
Users

Tags Page:
Tags

Unaswered Page:
Unaswered

Questions Page:
Questions

Reading Question:
Discord_EowNaQTQ8x


Search:
Search

**Comment by frozaum** [comment]
but for real, bas is a funny guy who doesn't know the difference of buzzfeed and buzzfeed news, so...

**Comment by Bas950** [comment]
> but for real, bas is a funny guy who doesn't know the difference of buzzfeed and buzzfeed news, so...

BLAME @eggsywashere

**Comment by frozaum** [comment]
> > but for real, bas is a funny guy who doesn't know the difference of buzzfeed and buzzfeed news, so...
> 
> BLAME @eggsywashere

IT WAS YOUR FAULT AND @eggsywashere FAULT AS WELL 😡

**Comment by eggsywashere** [comment]
> > but for real, bas is a funny guy who doesn't know the difference of buzzfeed and buzzfeed news, so...
> 
> BLAME @eggsywashere
YOU WERE THE ONE
image

**Comment by frozaum** [comment]
EXPOSED

**Comment by Bas950** [comment]
> > > but for real, bas is a funny guy who doesn't know the difference of buzzfeed and buzzfeed news, so...
> > 
> > 
> > BLAME @eggsywashere
> 
> YOU WERE THE ONE
> image

You confused me tho... Saying he already made it

**Comment by eggsywashere** [comment]
> > > > but for real, bas is a funny guy who doesn't know the difference of buzzfeed and buzzfeed news, so...
> > > 
> > > 
> > > BLAME @eggsywashere
> > 
> > 
> > YOU WERE THE ONE
> > image
> 
> You confused me tho... Saying he already made it

It was about the version stuff and you said why not make it in 1 lol, we confused each other #bestPR

**Comment by Bas950** [comment]
https://tenor.com/view/two-spiderman-pointing-meme-gif-11796793

**Comment by eggsywashere** [comment]
You mean ?
you-mean

**Comment by doomlerd** [comment]
Uh, guys? ehm.

File: BuzzFeed News/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      "Digital media",
+      "Viral Content",
+      "News",
+      "America New"
+   ],
+   "description": {
+      "en": "BuzzFeed, Inc. is an American Internet media, news and entertainment company with a focus on digital media; it is based in New York City. BuzzFeed was founded in 2006 by Jonah Peretti and John S. Johnson III, to focus on tracking viral content.",
+      "tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu."
+   },
+   "category": "others"
```
Comment (frozaum) [inline BuzzFeed News/dist/metadata.json]: just remove the "s"

Comment (Lanson2232) [reply BuzzFeed News/dist/metadata.json]: Done 👌

**Comment by frozaum** [comment]
The End.

**Pull request merged**